pages/home/glossary.mdx

```diff
@@ -30,6 +30,16 @@ Tools are commonly referred to by a qualified name that includes their toolkit.
 
 *Learn more about [tools](/home/build-tools/create-a-toolkit).*
 
+#### LLM-native tools
+
+[LLM-native tools](/home/use-tools/types-of-tools#llm-native-tools) are designed from scratch to provide the best performance for LLMs in terms of speed, reliability, accuracy, and cost-effectiveness.
+
+#### API wrapper tools
+
+[API wrapper tools](/home/use-tools/types-of-tools#api-wrapper-tools) are designed to mirror the original API design of the third-party service. They are not optimized for LLM usage and are not subject to evaluation suites. We recommend thoroughly evaluating each API wrapper tool with your Agents or chatbots before using it in production.
```

Comment:
I _think_ we've decided to converge on the term "upstream" instead of "third-party"

Reply:
updated